examples/text-generation/run_lm_eval.py

```diff
@@ -85,12 +85,20 @@ def setup_lm_eval_parser():
         help="Tasks to run",
         default=["hellaswag", "lambada_openai", "piqa", "winogrande"],
     )
+<<<<<<< HEAD
```

Comment:
@smarkovichgolan Could you please update your commit? There is an unresolved conflict in this file that needs to be addressed.